app/createTournament/createTournamentBusiness.js

```diff
@@ -0,0 +1,53 @@
+import * as common from '../../utils/common';
+import * as playerValidation from '../playersinscription/addPlayerBusiness';
+import {Player} from '../Player';
+import * as Math from '../../node_modules/mathjs/dist/math.min';
```

Comment:
There's already a native Math object in JS, so you likely don't need this one.
http://www.w3schools.com/js/js_math.asp